recipes/nvcomp/meta.yaml

```
@@ -0,0 +1,110 @@
+{% set version = "2.6.1" %}
+{% set cuda_major = (cuda_compiler_version|default("12.0.0")).split(".")[0] %}
+{% set platform = "x86_64" %}  # [linux64]
+{% set platform = "SBSA" %}  # [aarch64]
+{% set platform = "windows" %}  # [win]
+{% set extension = "tgz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: nvcomp
+  version: {{ version }}
+
+source:
+  - url: https://developer.download.nvidia.com/compute/nvcomp/{{ version }}/local_installers/nvcomp_{{ version }}_{{ platform }}_{{ cuda_major }}.x.{{ extension }}
+    sha256: 62046476076d1b455fb13fb78b38d3496bacafbfdbb0494fb3392240a04466ea  # [linux64 and cuda_major == "11"]
+    sha256: a0792b666caaf593a9dd4130979578fd3a78230f4407645c295700ef8e7aaaf2  # [linux64 and cuda_major == "12"]
+    sha256: a019f2c8a300991d94d718a3524377d978efadaa62349858741f17ef859a9e55  # [aarch64 and cuda_major == "11"]
+    sha256: fc668519a8344e9d05335bad4bc5d23a504cdc7579aea41f12d6aa0f3079e709  # [aarch64 and cuda_major == "12"]
+    sha256: 4f0105a51ac6ea947154e252c8b220baf3345803f55644b5aad84e22ae4f8986  # [win and cuda_major == "11"]
+    sha256: e362e1fbf6b101ad71aa5314f21d00a4353b90efbe5cdedc5fcfb0f3b68cd791  # [win and cuda_major == "12"]
+  # Include cudart license because it is statically linked.
+  - url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/LICENSE.txt
+    fn: cudart_LICENSE
+    sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
+
+build:
+  number: 0
+  # TODO: Enable aarch64 builds
+  # TODO: Enable Windows with CUDA 12 when it is supported
+  skip: True  # [win32 or osx or aarch64 or ppc64le or cuda_compiler_version in (undefined, "None", "10.2") or (win and cuda_major == "12")]
```

Since all CUDA 11 (and 12) versions are handled the same, think we can limit this to one of each

```suggestion
  skip: True  # [win32 or osx or aarch64 or ppc64le or cuda_compiler_version not in ("11.2", "12.0") or (win and cuda_major == "12")]
```